xbmc/windowing/X11/WinSystemX11.cpp

```diff
@@ -911,7 +911,8 @@ bool CWinSystemX11::CreateIconPixmap()
   gRatio = vis->green_mask / 255.0;
   bRatio = vis->blue_mask / 255.0;
 
-  CBaseTexture *iconTexture = CBaseTexture::LoadFromFile("special://xbmc/media/icon256x256.png");
+  CBaseTexture *iconTexture = CBaseTexture::LoadFromFile("special://xbmc/media/icon256x256.png",
+                                                         0, 0, "image/png");
```

Comment:
it might not be required any longer, i'll check. i suspect it's a leftover from my ffmpeg image stuff that you adopted.